web/elm/tests/RoutesTests.elm

```diff
@@ -68,7 +166,31 @@ all =
                     , fragment = Nothing
                     }
                     |> Expect.equal
-                        (Just (Routes.Dashboard { searchType = Routes.HighDensity, dashboardView = Routes.ViewNonArchivedPipelines }))
+                        (Just
+                            (Routes.Dashboard
+                                { searchType = Routes.HighDensity
+                                , dashboardView = Routes.ViewNonArchivedPipelines
+                                }
+                            )
+                        )
+        , test "dashboard hd view ignores search and instance group query params" <|
```

Comment:
It's mainly for simplicity of implementation, but also because it's hard to come up with a nice way to visualize them - the  `hyphen-notation` we use in tooltips could work, but it's not ideal (and as you note, not unique in all cases)